css/app.css

```diff
@@ -1,3 +1,448 @@
+.the-debate-vs::after {
+  content: "";
+  position: absolute;
+  height: 1px;
+  background-color: #d9d9d9;
+  display: block;
+  left: 0;
+  right: 0;
+  z-index: -1;
+}
+
+.the-debate-vs::before {
+  top: calc(50% - 2px);
+}
+
+.the-debate-vs::after {
+  top: 50%;
+}
+
+/* Medie queries */
+@media (min-width: 768px) {
+  .nav-button {
+    font-size: 18px;
+    padding: 0 15px;
+  }
+
+  .main-top-grid-container {
+    display: grid;
+    grid-template-columns: 45% 55%;
+    grid-template-rows: repeat(5, minmax(min-content, max-content));
+  }
+
+  .featured-stories {
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 4;
+  }
+
+  .top-story {
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 1;
+  }
+
+  .top-story-main-article {
+    border-bottom: 0;
+  }
+
+  .my-turn {
+    grid-column: 2 / span 1;
+    grid-row: 2 / span 1;
+  }
+
+  .my-turn-container,
+  .culture-travel-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 20px;
+  }
+
+  .culture-travel {
+    grid-column: 2 / span 1;
+    grid-row: 3 / span 1;
+  }
+
+  .more-stories {
+    grid-column: 2 / span 1;
+    grid-row: 4 / span 1;
+  }
+
+  .main-grid-container-aside {
+    grid-column: 1 / span 2;
+    grid-row: 5 / span 1;
+  }
+
+  .sign-up {
+    grid-column: 1 / span 2;
+    grid-row: 7 / span 1;
+  }
+
+  .magazine-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap: 30px;
+  }
+
+  .magazine-article-image {
+    overflow: hidden;
+  }
+
+  .magazine-article-image img {
+    height: 200px;
+  }
+
+  .title-logo {
+    border-bottom: 1px solid #ccc;
+  }
+
+  .magazine-article-title {
+    font-size: 20px;
+  }
+
+  .editor-picks-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 30px;
+  }
+
+  .editor-picks-title {
+    font-size: 24px;
+  }
+
+  .editor-picks-subtitle {
+    font-size: 16px;
+  }
+
+  .subscribe {
+    border-top: 1px solid rgba(218, 218, 218, 0.849);
+  }
+
+  .subscribe-title {
+    font-size: 24px;
+  }
+
+  .subscribe-details *,
+  .newsletter-header * {
+    text-align: left;
+  }
+
+  .newsletter-image {
+    width: 210px;
+    height: 254px;
+    overflow: hidden;
+  }
+
+  .newsletter-image img {
+    width: 210px;
+    height: 254px;
+  }
+
+  .featured-stories::before,
+  .the-debate::before,
+  .opinion::before,
+  .more-stories::before,
+  .my-turn::before,
+  .magazine::before,
+  .editor-picks::before,
+  .subscribe::before,
+  .newsletter::before,
+  .footer::before {
+    display: none;
+  }
+}
+
+@media (min-width: 992px) {
+  .nav-brand svg {
+    width: 396px;
+    height: 60px;
+  }
+
+  .nav-lg-middle {
+    font-size: 19px;
+  }
+
+  .nav-lg-middle-search {
+    line-height: 24px;
+    height: 24px;
+  }
+
+  .nav-lg-bottom {
+    color: #807f84;
+  }
+
+  .main-top-grid-container {
+    display: grid;
+    grid-template-columns: 30% 40% 30%;
+    grid-template-rows: repeat(4, minmax(min-content, max-content));
+  }
+
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.grid{
  display:grid;
}
```
**Html file**

```
<div class="grid">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_